modules/storage_account/storage_account.tf

```
@@ -32,6 +32,7 @@ resource "azurerm_storage_account" "stg" {
   location                          = local.location
   min_tls_version                   = try(var.storage_account.min_tls_version, "TLS1_2")
   is_hns_enabled                    = try(var.storage_account.is_hns_enabled, false)
+  sftp_enabled                      = try(var.storage_account.sftp_enabled, false)
```

Let's default to the provider's default with null. 

```suggestion
  sftp_enabled                      = try(var.storage_account.sftp_enabled, null)
```